sysdeps/dripos/meson.build

```diff
@@ -35,6 +35,7 @@ if not no_headers
 		'include/abi-bits/uid_t.h',
 		'include/abi-bits/wait.h',
 		'include/abi-bits/limits.h',
+		'include/abi-bits/utsname.h',
```

Comment:
was formatting changed here?

Reply:
I don't think so, at least it does not show in my editor, is that file not meant to use tabs?

Reply:
Disregard, fault of the GitHub app.